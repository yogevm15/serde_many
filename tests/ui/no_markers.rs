use serde_many::{DeserializeMany, SerializeMany};

#[derive(SerializeMany, DeserializeMany)]
#[serde_many()]
struct A;

fn main() {}
