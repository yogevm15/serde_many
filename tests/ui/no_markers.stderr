error: At least one marker is required (#[serde_many(... = "...")])
 --> tests/ui/no_markers.rs:3:10
  |
3 | #[derive(SerializeMany, DeserializeMany)]
  |          ^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `SerializeMany` (in Nightly builds, run with -Z macro-backtrace for more info)

error: At least one marker is required (#[serde_many(... = "...")])
 --> tests/ui/no_markers.rs:3:25
  |
3 | #[derive(SerializeMany, DeserializeMany)]
  |                         ^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `DeserializeMany` (in Nightly builds, run with -Z macro-backtrace for more info)
